copy_repos/copy_repos.sh

```diff
@@ -18,6 +18,6 @@ else
         mkdir -p $path_prefix/$platform_family/
         cp -r ~/repo/$repo_name/$box/* $path_prefix/$platform_family/
 	env > $path_prefix/$platform_family/dists/$platform_version/main/binary-"$arch"/build_info
-
 fi
 
+$HOME/build-scripts/copy_repos/generate_build_info_path.sh
```

Comment:
Please remove 
env > $path_prefix/$platform_family/dists/$platform_version/main/binary-"$arch"/build_info
env > $path_prefix/$platform/$platform_version/$arch/build_info
 above, they should be done using copy_repos/generate_build_info_path.sh